### DISCUSSION TITLE
How to search QA from JSON LINES file?

### DISCUSSION THREAD

**Discussion by liu-jin-lg** [FlowiseAI/Flowise#1229] [Q&A]
I need everyone's help.
I have a jsonl file with the content as
{"messages": [{"role": "user", "content": "xxxxx"}, {"role": "assistant", "content": "xxxxx"}]}
{"messages": [{"role": "user", "content": "xxxxx"}, {"role": "assistant", "content": "xxxxx"}]}
When I use Document Loaders: Json Lines File, I upload the file and enter "messages[].content" in Pointer Extraction.
No matter what I ask the AI, the AI's answer is "Error: Expected string, at position 0 got undefined".

I uploaded a picture of the configuration so that everyone can help me, thank you all
![image](https://github.com/FlowiseAI/Flowise/assets/149459619/390edac0-884b-4533-a996-b33945bcc5f9)

**Comment by HenryHengZJ** [comment]
you have to specify the pointer according to jsonpointer format - https://www.npmjs.com/package/jsonpointer. having said that im not sure how to specify the pointer syntax for nested object. I would probably convert the jsonl into a simple jsonl like: 
```
{"content": "This is a sentence."}
{"content": "This is another sentence."}
```
